Gruntfile.coffee

```diff
@@ -109,6 +109,7 @@ module.exports = (grunt) ->
           'build/js/components/*.js'
           'build/js/directives/*.js'
           'build/js/filters/*.js'
+          'build/js/helpers/*.js'
```

Comment:
When changing grunt like this, make sure to test the production build. Set `DIST=1` in `.env` and then:

```sh
rm -rf build dist
make server
```

Make sure to check at minimum the landing page and the login page.